config/channels/in-memory-channel/deployments/controller.yaml

```diff
@@ -73,6 +73,19 @@ spec:
 
         securityContext:
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          capabilities:
+            drop:
+            - all
+
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
```

Comment:
as said before, not sure we really have facts for bumping the numbers to this

Reply:
fwiw I think _any_ request is better than no request at all here, otherwise it really is just luck causing the controller to be on a node with any mem/cpu left at all iyswim

Reply:
These requests were copied from serving; I agree that they could probably be tuned (lower, I'm hoping), but leaving them out entirely doesn't do the scheduler any favors (and just pushes the issue to any responsible downstream distributors).

I agree that it would have been good if this had been in earlier in the release cycle; I forgot about it until 2 days ago.